docs/sources/alerting/unified-alerting/contact-points.md

```diff
@@ -54,20 +54,20 @@ VictorOps | `victorops`
 [Webhook](#webhook) | `webhook`
 [Zenduty](#zenduty) | `webhook` 
 
-## Manage contact points for an external Alertmanager
+## Managing contact points for an external alertmanager
```

Comment:
Please keep "Manage". 
Also, I thought "Alertmanager" is a feature and we need to keep it with uppercase A?